bool a
---
int64 n_rows
int64 n_cols
float64[] data